static/styles/user_circles.css

```diff
@@ -27,12 +27,13 @@ $idle_color: hsl(29, 84%, 51%);
 }
 
 .user_circle_empty {
-    background-color: none;
-    border-color: hsl(0, 0%, 50%);
+    background-color: hsl(54, 31%, 75%);
+    border-color: hsl(54, 31%, 75%);
 }
 
 .user_circle_empty_line {
-    border-color: hsl(0, 0%, 50%);
+    border-color: hsl(54, 31%, 75%);
+    background-color: hsl(54, 31%, 75%);
```

Comment:
Is this changing the colors to match design proposals?   Let's not do that here -- we want this PR to be self-contained.

Reply:
Yes. How to make this PR to be self-contained?